src/components/dev-hub/podcast-player/audio-player.js

```diff
@@ -0,0 +1,236 @@
+import React, { useCallback, useReducer, useRef } from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import { css } from '@emotion/react';
+import ReactPlayer from 'react-player/lazy';
+
+import PlayerControls, {
+    ControlsContainer,
+} from '~components/dev-hub/podcast-player/player-contols';
+import PlayerListenOn, {
+    LogosContainer,
+} from '~components/dev-hub/podcast-player/player-listen-on';
+import Slider, { SliderContainer } from '~components/dev-hub/slider';
+
+import { getTimeLabel } from '~utils/get-time-label';
+import { size, colorMap } from '~components/dev-hub/theme';
+
+import {
+    PLAYER_ACTIONS,
+    playerInitialState,
+    playerReducer,
+} from '~components/dev-hub/podcast-player/player-reducer';
+
+const APPLE_PODCASTS_URL =
+    'https://podcasts.apple.com/us/podcast/the-mongodb-podcast/id1500452446';
+const SPOTIFY_PODCASTS_URL =
+    'https://open.spotify.com/show/0ibUtrJG4JVgwfvB2MXMSb?si=k1oOQ8JcSr6WtfeSMQ-kFA&nd=1';
+const GOOGLE_PODCASTS_URL =
+    'https://podcasts.google.com/feed/aHR0cHM6Ly9tb25nb2RiLmxpYnN5bi5jb20vcnNz?sa=X&ved=2ahUKEwiE97rnkfDrAhU1n3IEHanUCRUQ9sEGegQIARAC';
```

Comment:
We should just move these to the player on component since that is the only place where used. Does not make sense to have the props just for these as well.